app/views/layouts/application.html.erb

```diff
@@ -41,7 +41,14 @@
     } %>
 
     <% if yield(:back_link).present? %>
-      <%= render "govuk_publishing_components/components/back_link", href: yield(:back_link) %>
+      <% if yield(:back_link).include?('text') %>
```

Comment:
I think we could have a problem here if a link happens to include the word 'text' - e.g. a title. Perhaps it's safer to just have a ':custom_back_link' content block, so it doesn't interfere with the current one?

Reply:
👍